pkg/server/api/middleware/logger.go

```diff
@@ -26,7 +26,9 @@ func NewRequestLogger(logger log.Logger) chain.Middleware {
 			recorder := httptest.NewRecorder()
 
 			scopedLogger := logger.
-				With("http_request", r)
+				With("headers__host", r.Header.Get("Host")).
+				With("headers__content_type", r.Header.Get("Content-Type")).
+				With("headers__user_agent", r.Header.Get("User-Agent"))
```

Comment:
I think potentially we want to annotate this `scopedLogger` with the method and path as well (or potentially _instead of_ where we do it below)?

e.g. where we currently log `msg="Creating instance"`, at the moment we're dumping out the whole `http_request` object (which is obviously bad), but after this change we'd be missing some of this potentially useful info.

Reply:
These headers may be useful too:
- `X-Forwarded-For`
- `X-Cloud-Trace-Context`
- `Draupnir-Version`